lib/src/modules/conditional.dart

```diff
@@ -928,6 +928,13 @@ class If extends Conditional {
   /// If [condition] is 1, then [then] executes, otherwise [orElse] is executed.
   If(this.condition, {this.then = const [], this.orElse = const []});
 
+  /// If [condition] is 1, then [sig] is excutes.
+  ///
+  /// Use this constructor when you only have a single condition.
+  If.s(this.condition, Conditional sig, [this.orElse = const []]) {
```

Comment:
Let's change it from `sig` to `then` so that it's more clear that it's the same thing.

The `orElse` should also be just one `Conditional` rather than a list

Rather than making `then` `late`, you can do `If.s(...) : this(condition, then: [then], orElse: [orElse] )`